### PR TITLE
Checkpoint Review

### PR DESCRIPTION
**Pull request by andela-vkironde**

### CODE REVIEW

File: app/assets/stylesheets/flight.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Place all the styles related to the Flight controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.article-title {
```
Comment (Habu-Kagumba) [inline app/assets/stylesheets/flight.scss]: Please indent file appropriately.

Comment (Habu-Kagumba) [inline app/assets/stylesheets/flight.scss]: This applies to other styles files. (.scss files)

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.find_by_email(params[:session][:email])
```
Comment (Habu-Kagumba) [inline app/controllers/sessions_controller.rb]: You should use strong parameters provided by rails.

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module UsersHelper
+  def booking_expired(booking)
+    p booking.flight.departure_date(booking.flight.departure)
```
Comment (Habu-Kagumba) [inline app/helpers/users_helper.rb]: What is the use for `p` here?

File: app/models/airline.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class Airline < ApplicationRecord
+  has_many :flights
+ validates :code, presence: true, uniqueness: true
```
Comment (Habu-Kagumba) [inline app/models/airline.rb]: Fix indentation

File: coverage/.last_run.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Habu-Kagumba) [inline coverage/.last_run.json]: Ignore the `coverage/` dir

File: spec/controllers/bookings_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,288 @@
+require 'rails_helper'
+
+RSpec.describe BookingsController, type: :controller do
+
+  subject(:booking) { create(:booking) }
+  let(:flight) { create(:flight) }
+  let(:passenger) { create(:passenger)}
+  before(:each) do
```
Comment (Habu-Kagumba) [inline spec/controllers/bookings_controller_spec.rb]: Remove this empty block.

Comment (Habu-Kagumba) [inline spec/controllers/bookings_controller_spec.rb]: Also there is a lot of repeatition on this file. Please try to refactor.

File: spec/controllers/bookings_controller_spec.rb [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+        stub_current_user(booking.user)
+        patch :update, params: {
+          id: booking.id,
+          booking: attributes_for(
+            :booking,
+            email: booking.email,
+            departure: booking.departure,
+            flight_id: booking.flight.id,
+            user_id: booking.user_id,
+            passengers: attributes_for(:passenger)
```
Comment (Habu-Kagumba) [inline spec/controllers/bookings_controller_spec.rb]: The booking controller permits `passengers_attributes` and not `passengers`

File: app/assets/javascripts/bookings.coffee [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (akabiru) [inline app/assets/javascripts/bookings.coffee]: Consider deleting unused files.

File: app/assets/stylesheets/pages.scss [hunk 9]
```diff
@@ -1,3 +1,98 @@
...
+  height: 90%;
+ 	background-repeat: no-repeat;
+ 	background-color: #d3d3d3;
+ 	font-family: 'Oxygen', sans-serif;
+  background-image: url("bg.jpg");
+  background-size: 100%,100%;
+  background-attachment: fixed;
+
+}
+body { padding-top: 70px; }
```
Comment (akabiru) [inline app/assets/stylesheets/pages.scss]: Indent appropriately.
```css
body {
  padding-top: 70px;
}
```

Fix for all instances in this  file.

File: bin/setup [hunk 10]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env ruby
```
Comment (akabiru) [inline bin/setup]: Exclude `bin` folder in `rubocop.yml`

File: config.ru [hunk 11]
```diff
@@ -1,5 +1,5 @@
 # This file is used by Rack-based servers to start the application.
```
Comment (akabiru) [inline config.ru]: Exclude all rails generated files in rubocop

File: bin/setup (hunk 10 above)
Comment (andela-vkironde) [inline bin/setup]: done

File: config.ru (hunk 11 above)
Comment (andela-vkironde) [inline config.ru]: done

**Pull request merged**